### ISSUE TITLE
[FEATURE REQUEST] Light Gun support

### ISSUE THREAD

**Issue by RGarciaLago** [MiSTer-devel/Saturn_MiSTer#169]
Would be great to get Light Gun support to be able to use it outside of SNAC and a CRT.

There are 15 games that support it:
- Area 51
- Chaos Control Remix
- Crypt Killer
- Daisuki
- Death Crimson
- Die Hard Trilogy
- The House of the Dead
- Jinzou Ningen Hakaider: Last Judgement
- Kochira Katsushikaku Kamearikouenmae Hashutsujo Nakagawa Land Dai Race! no Maki
- Maximum Force
- Mighty Hits
- Policenauts
- Scud: The Disposable Assassin
- Virtua Cop
- Virtua Cop 2

**Comment by gleek12** [comment]
Having it support the Wii remote similar to how the psx core handles it would be amazing!  Thanks for you hard work!

**Comment by ajgowans** [comment]
It would be fantastic to play House Of The Dead with a WiiMote.

Worth noting lightgun support is seemingly not implemented on SNAC either at present, so needs support added there also.

**Comment by ucbsuperfreak** [comment]
Ya if we could get SNAC support for light gun that would be phenomenal. I have the Virtua Cop light gun for Saturn waiting for action.

Thank you to everyone who works to help develop these cores. It's so cool watching them grow in functionality and performance.

**Comment by JaeOfMordor** [comment]
time to close this one @RGarciaLago

**Comment by JaeOfMordor** [comment]
@ucbsuperfreak time to break out your virtua gun

**Comment by RGarciaLago** [comment]
> time to close this one @RGarciaLago

I appreciate the ping! I just checked and light gun support is still SNAC only so I’ll keep the issue open until it’s been implemented for USB light guns as well.

**Comment by ucbsuperfreak** [comment]
@JaeOfMordor this is awesome, thank you so much! Do I just have to do a regular update all to get the core with this implemented?

**Comment by JaeOfMordor** [comment]
no, the quickest way is to go onto the official mister discord and get it from the unstable-nightly tab

**Comment by Kuba-J** [comment]
> @JaeOfMordor this is awesome, thank you so much! Do I just have to do a regular update all to get the core with this implemented?

Link to the trst core:
https://github.com/MiSTer-unstable-nightlies/Saturn_MiSTer/releases/download/unstable-builds/Saturn_unstable_20240929_15da53.rbf

**Comment by ucbsuperfreak** [comment]
Hi I finally got around to trying this and am not able to get the light gun calibrated/working. When I try the calibration tool using F10, nothing comes up. And I don’t see a lightgun option in the input configuration. Sorry to be a pain, but could you walk me through what I’m doing wrong? 

I’m using the most recent unstable nightly. Thanks for your help!

**Comment by JaeOfMordor** [comment]
@ucbsuperfreak 
>Sorry to be a pain, but could you walk me through what I’m doing wrong?

basically everything.

literally just make sure you’re running the latest unstable-nightly release, plug in your sega saturn Virtua Gun to a basic snac adapter, turn on snac in the menu, play.

if you bought some weird proprietary adapter or try to use a ps1 gun it’ll do nothing.

**Comment by foxp4v** [comment]
Hope to be able to use my Namco GunCon2 as in the other cores that support It.

**Comment by RGarciaLago** [comment]
Thank you @JaeOfMordor for notifying of the great news! Thanks for all your awesome work on this @ElectronAsh and @srg320! Closing issue.
